app/src/components/custom-radio-buttons.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Icon, useRadio, UseRadioProps } from "@chakra-ui/react";
+import { InventoryButtonCheckIcon } from "./icons";
+
+interface CustomRadioProps extends UseRadioProps {
+  children: React.ReactNode;
+}
+
+export function CustomRadio(props: CustomRadioProps) {
```

Comment:
Can you make sure the file name and component match?
So renaming the file to custom-radio-button.tsx and the component to CustomRadioButton perhaps.